template/config/index.js

```diff
@@ -1,13 +1,30 @@
 // see http://vuejs-templates.github.io/webpack for documentation.
 var path = require('path')
+{{#cdn}}
+var dependencies = require('../package.json').dependencies;
+var packages = [{
+  name: 'vue',
+  var: 'Vue',
+  path: 'dist/vue.runtime.min.js'
+}]
+
+if (dependencies['vue-router'] || dependencies.vuex) {
```

Comment:
yes, but here you're adding vue-router if vuex is in the dependencies. It could be something like:

```js
var cdnPackages = [{
    name: 'vue-router',
    var: 'VueRouter',
    path: 'dist/vue-router.min.js'
}, {
    name: 'vuex',
    var: 'Vuex',
    path: 'dist/vuex.min.js'
}]

packages = packages.concat(cdnPackages.filter(({ name }) => name in dependencies))
```